script.js

```diff
@@ -0,0 +1,139 @@
+let files;
+let currSection = "";
+const hostURL = 'https://api.jsonbin.io/v3/b/62fa3b83a1610e6386ff2396/';
+
+const loadContent = async () => {
+
+    let coursesSections = ["data-science", "drawing", "aws", "excel", "js", "python", "web-dev"];
```

Comment:
declare the data that won't be changed with const instead of let
```suggestion
    const coursesSections = ["data-science", "drawing", "aws", "excel", "js", "python", "web-dev"];
```